bve/src/parse/mesh/instructions/post_processing.rs

```diff
@@ -1,8 +1,22 @@
 use crate::parse::mesh::instructions::*;
-use crate::parse::mesh::{MeshError, MeshErrorKind, Span};
+use crate::parse::mesh::{MeshError, MeshErrorKind};
+use crate::parse::Span;
 use cgmath::{Array, Vector2, Vector3};
 use std::f32::consts::PI;
 
+/// Prepares instructions for execution.
+///
+/// Performs two postprocessing steps on the instruction list:
+///
+/// - Splits up [`Cube`] and [`Cylinder`] into their respective [`AddVertex`] and [`AddFace`] instructions. This allows
+///   the following:
```

Comment:
What following does it allow?

Reply:
The next postprocess step, will make that more clear.